index.html

```diff
@@ -591,6 +591,22 @@ <h1 class="section-h1"><span class="white-border">Student Games</span></h1>
           <div class="container">
             <section>
               <div class="row">
+		<div class="col-xs-4 col-sm-4 col-md-3 col-lg-2">
+                  <div class="card">
+                    <a href="https://github.com/Juanvulcano/GCI/tree/master/Fossasia/Week1/Treasurehunt">
+                      <div class="avatar img-circle">
+                        <img class="card-img-top" src="img/gallery/Treasurehunt.png" data="img/gallery/Treasurehunt.png" alt="Juanvulc$
```

Comment:
@Juanvulcano @jig08 Why is there a Dollar sign `$` here instead of a closing tag? :wink: